src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -815,7 +843,7 @@ public boolean accept(
                     // TODO ask for independent frame if we're skipping a TL0.
 
                     int sourceIdx = sourceFrameDesc.getRTPEncoding().getIndex();
-                    if (sourceEncodings[currentIdx].requires(sourceIdx))
+                    if (sourceEncodings[this.currentIdx].requires(sourceIdx))
```

Comment:
`this.currentIdx` has already been updated to the new value of `newCurrentIdx` by here, so i think this should be alright?